### ISSUE TITLE
Make code to go through parameter space

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/temporalvar#11]
Make code to go through parameter space: Inter and intra specific competition (alphas) and tauIs.

**Comment by lizzieinvancouver** [comment]
Playing around ....
Okay, I played around a little (whee! Iove playing around) -- enough that the next task is formally coding up some loops to loop through parameters ... it's sunny so I think I may leave this as a tomorrow/next day task.

But -- good news! It's quite easy to make the storage effect ameliorate a lambda difference. Try this:
tauI <- c(0.4, 0.4)
lambda <- c(5,6)

If you run it species 2 wins as expected (also, oddly the bar chart shows equal effects of the storage effect and pc growth rate -- this seems especially buggy). Then try:

tauI <- c(0.46, 0.4)
lambda <- c(5,6)

And voila, species 1 cycles and stays in the game (for most runs, I think I saw 1/5 where it did not). For more consistent results try:
tauI <- c(0.47, 0.35)
lambda <- c(5,6)

Somewhere around:
tauI <- c(0.47, 0.1)
lambda <- c(5,6)
You see species 2 go extinct, sometimes.
